etl/RetrosheetTableBuilder.py

```diff
@@ -6,17 +6,15 @@
 import os
 from os import listdir
 from os.path import isfile, join
-onlyfiles = [f for f in listdir(mypath) if isfile(join(mypath, f))]
 import sys
 
 import psycopg2
 
 from RetrosheetParser import RetrosheetLineParser
 
-# Previously read class RetrosheetTableBuilder()...shouldn't there be 'object' inside () and ":" at the end of the line?
```

Comment:
Yes - good catch. The 'object' inheritance is good protection to make sure regardless of whether someone is running with Python 2 or 3 everything will use the new-style Python classes.